vyper/context/types/function.py

```diff
@@ -279,19 +284,17 @@ def from_FunctionDef(
             raise StructureException("Cannot use reentrancy guard on view or pure functions", node)
 
         # call arguments
-        arg_count: Union[Tuple[int, int], int] = len(node.args.args)
+        min_arg_count = len(node.args.args)
+        max_arg_count = len(node.args.args)
         if node.args.defaults:
             if node.name == "__init__":
```

Comment:
can this use `node.is_constructor` property?

Reply:
no, because this is happening inside a class method we aren't able to use that function